src/frontend/bundle-entry.jsx

```diff
@@ -21,8 +21,14 @@ const env = {
 
 const store = createStore(window.REDUX_STATE)
 const theme = window.THEME
+const adminLocale = window.REDUX_STATE.locale
```

Comment:
Is it safe to access `window.REDUX_STATE` instead of `useSelector`?

CC @ariansobczak-rst

Reply:
This variable is outside store provider so the store is here equivalent to `window.REDUX_STATE` or `store.getState()`